aprobacion

jdiazg0312 opened the pull request:
ve lo que envio es poco xq primero me quiero asegurar de que lo estoy haciendo bien... inserte la de mostrar la lista por estado, espero tu aprobacion para seguir con las demas..

AlexVillegasC committed:
Merge pull request #2 from AlexVillegasC/estebancas-master

Estebancas master

AlexVillegasC committed:
Delete README.md

AlexVillegasC committed:
Readme

# UNA.IngSistemas.Aprobe
Desarrollo de un software para el MEP, para agilizar el proceso de solicitud de beca.

AlexVillegasC committed:
Merge pull request #3 from AlexVillegasC/estebancas-master

Documentos Casos de uso y Requerimientos

jdiazg0312 committed:
agragado

agregue un caso de uso de mostrar lista de estado en el pull te dije
unas cosas

AlexVillegasC commented:
Comentame que fue lo que agragastes!, lo revise pero no vi cambios a grandes rasgos

jdiazg0312 committed:
flujo alterno agregado

puse los flujos alternos, tambien creo a mi punto de vista que algunos
no deben de tener flujos alternos si me equivoco notifiquenlo

alexvillegasopenpath commented:
![MuaKissGIF](https://github.com/user-attachments/assets/cb5fc998-7858-4568-82ae-4f12835fd352)
